src/components/header-block/header-block.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { down, up } from '../breakpoint/breakpoint';
+
+const TopLineR = css`
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 110%;
+`;
+
+const HeadlineXL = css`
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 110%;
+`;
+
+const TextS = css`
+  font-size: 14px;
+  line-height: 150%;
+`;
+
+const TextL = css`
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 150%;
+`;
+
+const Text = css`
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 150%;
+`;
+
+const BlockWrapper = styled.div`
+  display: flex;
+  ${down('sm')} {
+    flex-direction: column;
+  }
+`;
+
+const TextWrapper = styled.div`
+  display: inline;
+`;
+
+const IconWrapper = styled.div`
+  width: 144px;
+  height: 144px;
+  ${down('sm')} {
+    margin-left: auto;
+  }
+`;
+
+const Topline = styled.p`
+  ${TopLineR}
+  color: #3e61ee;
+  margin: 0;
+`;
+
+const HeaderBlock = (props) => {
+  const HeaderBlockText = styled.p`
+    ${props.large ? TextL : Text}}
+    letter-spacing: -0.01em;
+    color: #202840;
+    margin-top: 32px;
+  `;
+
+  const Headline = styled.p`
+    ${HeadlineXL}
+    letter-spacing: -0.01em;
+    color: #202840;
+    margin-top: 16px;
+    margin-bottom: ${props.large ? '32px' : '16px'};
+  `;
+
+  const Timestamp = styled.p`
+    ${TextS}
+    letter-spacing: -0.01em;
+    color: rgba(0, 0, 0, 0.5);
+    margin-top: ${props.large ? '32px' : '16px'};
+    margin-bottom: 32px;
+  `;
+
```

Comment:
always define them outside. this should cause a warning in the console, maybe you have missed it?

Reply:
I started with having them outside, but then I could not access the props values inside the css. Is there another way to do this?

Reply:
you can pass props to styled components, here a simplfied exampel taken from the repo somewhere (search to find the place. there are plenty of other examples like this. `styled.div<>` contains the prop definition for this specific styled component to tell what's being passed in. You can try it to define but if you struggle remove it (defaults to any I think)

```
<AuroraBackground source={getSource(type)} />

const AuroraBackground = styled.div<AuroraBackgroundProps>`
  background-image: url(${(props) => props.source});
`;

```